breathecode/marketing/tests/tasks/tests_persist_single_lead.py

```diff
@@ -642,3 +648,316 @@ def test_persist_single_lead_with_form_entry_with_data(self):
+                     'automation': model['automation'].acp_id
+                 }}),
+            call('POST',
+                 'https://old.hardcoded.breathecode.url/api/3/contactTags',
+                 headers={
+                     'Accept': 'application/json',
+                     'Content-Type': 'application/json',
+                     'Api-Token': model['active_campaign_academy'].ac_key
+                 },
+                 json={'contactTag': {
+                     'contact': 1,
+                     'tag': model['tag'].acp_id
+                 }})
+        ])
+
+    # def test_register_new_lead_without_form_data(self):
+
+    #     model = self.generate_models(
+    #         academy=True,
+    #         active_campaign_academy=True,
+    #         tag=True,
+    #         tag_kwargs={'tag_type': 'STRONG'},
+    #         automation=True,
+    #         automation_kwargs={'slug': 'they-killed-kenny'},
+    #         form_entry=True,
+    #         form_entry_kwargs=generate_form_entry_kwargs(),
+    #         active_campaign_academy_kwargs={'ac_url': 'https://old.hardcoded.breathecode.url'})
+
+    #     data = generate
+    #     entry = register_new_lead()
+
+    #     self.assertEqual(entry, {})
+    #     self.assertEqual(self.all_form_entry_dict(), [])
+
+    # @patch(GOOGLE_CLOUD_PATH['client'], apply_google_cloud_client_mock())
+    # @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    # @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    # @patch(MAILGUN_PATH['post'], apply_mailgun_requests_post_mock())
+    # @patch(OLD_BREATHECODE_PATH['request'], apply_old_breathecode_requests_request_mock())
+    # @patch(
+    #     REQUESTS_PATH['get'],
+    #     apply_requests_get_mock([(
+    #         200,
+    #         f'https://maps.googleapis.com/maps/api/geocode/json?latlng=15.000000000000000,15.000000000000000&key={GOOGLE_CLOUD_KEY}',
+    #         {
+    #             'status':
+    #             'OK',
+    #             'results': [{
+    #                 'address_components': [{
+    #                     'types': {
+    #                         'country': 'US',
+    #                     },
+    #                     'long_name': 'US',
+    #                 }, {
+    #                     'types': {
+    #                         'locality': 'New York',
+    #                     },
+    #                     'long_name': 'New York',
+    #                 }, {
+    #                     'types': {
+    #                         'route': 'Avenue',
+    #                     },
+    #                     'long_name': 'Avenue',
+    #                 }, {
+    #                     'types': {
+    #                         'postal_code': '10028'
+    #                     },
+    #                     'long_name': '10028',
+    #                 }]
+    #             }]
+    #         })]))
+    # def test_persist_single_lead_with_form_entry_with_data_current_download(self):
```

Comment:
Remove it, also is repeated `test_persist_single_lead_with_form_entry_with_data_current_download` and `test_persist_single_lead_with_form_entry_with_data__current_download`